hvpy/api_groups/jpeg2000/tests/test_get_jp2_header.py

```diff
@@ -1,16 +1,21 @@
 import pytest
 
-from hvpy import getJP2Header
+from hvpy import getJP2Header, set_api_url
 from hvpy.api_groups.jpeg2000.get_jp2_header import getJP2HeaderInputParameters
 
 
 def test_getJP2HeaderInputParameters():
-    response = getJP2Header(id=9838343)
+    # Because the hvpy is testing the beta version of the API,
+    # the URL is set to the beta version in the first test case.
+    # So that all the following test cases will use this URL.
+    set_api_url("https://api.beta.helioviewer.org/v2/")
```

Comment:
Without this, pytest tests in default API which causes fails, and the test passes in tox because `environment var` has beta URL. So setting the API URL to beta URL in the first test case changes the URL for all the following tests.

Reply:
I'm sorry, I don't understand. We set the URL to be beta, why do we need to set it again?

Reply:
We set the URL only in the tox environment, but whenever we run the test cases using pytest (`pytest -c hvpy`) the test fails.

Reply:
Yes you have to set the URL for the tests to pass now. That's expected and this should be removed.

Reply:
This information should also be added to the dev guide page on testing before we forget

Reply:
Should we include updating docs in this PR?

Reply:
Yes please.